docs/settings.md

```diff
@@ -0,0 +1,204 @@
+# Settings
+
+This page will cover the Settings sections.  
+Each setting type is grouped based on what they're related to.
+
+## Media Management
+
+Here is where you configure your settings related to how your media files are handled.
+
+### Naming
+
+#### Volume Folder Naming
+
+The naming format for the volume folder (folder structure allowed).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year})` will come out as "Invincible Iron Man (2019)".
+
+#### File Naming
+
+The naming format for the file itself.  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+- {issue_comicvine_id}
+- {issue_number}
+- {issue_title}
+- {issue_release_date}
+- {issue_release_year}
+
+For example, `{series_name} ({year}) Volume {volume_number} Issue {issue_number}` would come out as "Invincible Iron Man (2019) Volume 01 Issue 001.cbz".
+
+#### File Naming For TPB
+
+The naming format for the file itself (if it's a TPB).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year}) Volume {volume_number} TPB` would come out as "Invincible Iron Man (2019) Volume 01 TPB.cbz"
+
+#### Issue Padding
+
+This defines how issue numbers are 'padded' (3 digits, 2 digits, etc.).  
+This is useful in the case of file sorting, as many operating systems sort _alphabetically_, instead of alphanumerically. To them, because 1 is lower than 2, "11" comes before "20".  
+To get around this, we 'pad' the numbers with leading zeroes. "1" becomes"001", "11" becomes "011", and "2" becomes "002" - resulting in a more accurate sort.
+
+Options are:
+
+- x (no padding, i.e. 1,2,...10)
+- 0x (2 digits, i.e. 01, 02...10)
+- 00x (3 digits, i.e. 001, 002...010...099,100)
+
+#### Volume Padding
+
+This defines how volume numbers are 'padded' (3 digits, 2 digits, etc.).  
+This is useful in the case of file sorting, as many operating systems sort _alphabetically_, instead of alphanumerically. To them, because 1 is lower than 2, "11" comes before "20".  
+To get around this, we 'pad' the numbers with leading zeroes. "1" becomes"001", "11" becomes "011", and "2" becomes "002" - resulting in a more accurate sort.
+
+Options are:
+
+- x (no padding, i.e. 1,2,...10)
+- 0x (2 digits, i.e. 01, 02...10)
+- 00x (3 digits, i.e. 001, 002...,010...099,100)
+
+### Unzipping
```

Comment:
Maybe include a sort of warning that Kapowarr deletes files that it thinks are irrelevant. If it thinks that a file in the zip file is not for the volume, it will not extract it (and thus will be deleted when the zip file is deleted). Also maybe say that the zip file will be deleted afterwards. And that if someone finds faulty behaviour in what Kapowarr deletes for a volume, that they should make an issue for it.

Reply:
Added.